m4_sessions/1803-inning/group_4.md

```diff
@@ -1,13 +1,46 @@
-Group 4
+## Group 4 Finding Public Data and Using it to Change the World
 
-Group Members:
+### Group Members:
+- Nora Gully
+- Kelley Jenkins
+- Casey Dallavalle
+- Josi McClellan
 
-Name of Session:
+### Name of Session: 
+- Finding Public Data and Using it to Change the World
 
-Session Description (Why should students come to this session):
+### Session Description (Why should students come to this session):
+- To learn about publicly available APIs that they can use with on their personal projects.
 
-Session Outcomes (What will students learn/leave with?):
+### Session Outcomes (What will students learn/leave with?):
+- A list of public apis.
+- Ideas for apps they can build with public data to solve real world problems.
+- AN IDENTITY AS A SOCIAL JUSTICE WARRIOR!
+- A sense of civic duty.
+
+
+### Session Outline (Please include who is facilitating and how much time you are going to spend in each section):
```

Comment:
EXCELLENT outline.  Love that you have identified timing and who is facilitating.  It would be awesome to have a slack channel where the groups could share what they found and follow up on projects they put together.

Reply:
There's a #data channel that pretty much covers this, would another channel be redundant?